src/source/PeerConnection/Rtcp.c

```diff
@@ -2,31 +2,125 @@
 
 #include "../Include_i.h"
 
+STATUS findTransceiverBySsrc(PKvsPeerConnection pKvsPeerConnection, PKvsRtpTransceiver* ppTransceiver)
```

Comment:
Can we assume the parameters are sanitized for this call?

Reply:
good question
i dont think we should assume that